common/forks/forks_test.go

```diff
@@ -17,16 +17,19 @@ func TestCollectSortedForkBlocks(t *testing.T) {
 		ConstantinopleBlock: big.NewInt(0),
 	})
 	require.Equal(t, l, []uint64{
+		0,
 		3,
 		4,
 	})
 	require.Equal(t, map[uint64]bool{
 		3: true,
 		4: true,
+		0: true,
 	}, m)
 	require.Equal(t, map[string]uint64{
-		"eip155": 4,
-		"eip158": 3,
+		"eip155":         4,
+		"eip158":         3,
+		"constantinople": 0,
```

Comment:
is it intentional that byzantium is not present in this map?

Reply:
as before, it's intentional

Reply:
Suggestion: https://github.com/scroll-tech/scroll/pull/1205

Reply:
merged